gcp_variant_transforms/libs/vcf_field_conflict_resolver_test.py

```diff
@@ -34,40 +37,164 @@ def _field_count(self, symbol):
     # symbol = {'A', 'G', 'R'}.
     return vcf.parser.field_counts[symbol]
 
-  def test_resolving_field_definition_conflict_in_type(self):
+  def test_resolving_schema_conflict_type(self):
+    test_data_configs = [
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
```

Comment:
nit: Since the structure of this config is shared among methods, I think it's better to create a [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) instead. So, it would be something like:
```
SchemaTestConfig = namedtuple('SchemaTestConfig', ['schema_type', 'schema_mode', etc])
test_configs = [
  SchemaTestConfig(BOOLEAN, NULLABLE, 1, True),
  # or if you want it to be more readable as:
  SchemaTestConfig(schema_type=BOOELAN, schema_mode=NULLABLE, etc),
  ...
]
```

Reply:
Done!